resources/views/components/productlist.blade.php

```diff
@@ -2,67 +2,78 @@
 
 @if ($value)
     <lazy v-slot="{ intersected }">
-        <listing v-if="intersected">
-            <x-rapidez::reactive-base>
-                <reactive-list
-                    component-id="{{ md5(serialize($value)) }}"
-                    data-field="{{ $field }}"
-                    :size="999"
-                    :default-query="function () { return { query: { terms: { '{{ $field }}': {!!
-                        is_array($value)
-                            ? "['".implode("','", $value)."']"
-                            : $value
-                    !!} } } } }"
+        <listing v-if="intersected" v-cloak inline-template>
```

Comment:
What's the alternative for this as `inline-template` isn't available anymore in Vue 3? @indykoning